gamepad_controller:
  ros__parameters:
    subscribed_axes:
      - "axis_0"
      - "axis_1" 
      - "axis_2"
      - "axis_3"
    subscribed_buttons:
      - "button_0"
      - "button_1"
      - "button_2"
      - "button_3"